python-main.js

```diff
@@ -1737,10 +1737,12 @@ function web_editor(config) {
         if (links) {
             Object.keys(links).forEach(function(key) {
                 if (links[key] === "close") {
-                    modalLinks.push('<a href="#" id="modal-msg-close-link">' + key + '</a>');
+                    modalLinks.push('<button type="button" area-labelledby="modal-msg-close-link" id="modal-msg-close-link">' + key + '</button>');
+
                     addCloseClickListener = true;
                 } else {
-                    modalLinks.push('<a href="' + links[key] + '" target="_blank">' + key + '</a>');
+                    modalLinks.push('<button type="button" aria-label="' + key + '"class="button-link">' + key + '</button>');
+                    modalLinks.onclick=function(){"window.location.href(links[key])"};
```

Comment:
Have you tested this? I think this has broken the links in the WebUSB not available modal.
Also, would this not open the link in the same window instead of a new one?

Reply:
Got a bit stuck triggering the button, but have fixed this and it should now work